views/introduction.pug

```diff
@@ -28,7 +28,9 @@ html(lang='en')
 
     block stylesheets
       link(rel='stylesheet', href='css/bulma.min.css', type='text/css', media='all')
+
     link(rel='prefetch', href='/index.html')
+    link(rel='shortcut icon' href='https://cdn.auth0.com/website/webauthn/favicon.png')
```

Comment:
I can't see on heroku review app favicon

Reply:
Yes I am reading pug docs to see if there is a specific block we need to add it to